check_run/public/js/custom/payment_entry_custom.js

```diff
@@ -47,3 +53,28 @@ function load_supplier_default_mode_of_payment(frm) {
 				})
 		})
 }
+
+function setup_remove_from_check_run(frm) {
+	if (frm.doc.docstatus > 0 && frm.doc.check_run && frm.doc.bank_account) {
+		let _doc = frm.doc
+		_doc.pay_to_account = frm.doc.paid_to
```

Comment:
What is this for?

Reply:
Payment Entry and Check Run have different fieldnames that link to the same thing, this is get it to match that features the function is expecting. We could change it on the server side where there would by typing and it'd be clearer.